src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -286,6 +286,14 @@ private void getChanges2(ChangeEventSourceContext context,
             bootstrapTabletWithRetry(tabletPairList);
         }
 
+        // Launch a thread to poll for the tables.
```

Comment:
An executor service should be used instead. Specifically a fixed thread pool. Also code to shutdown the thread is required.

Reply:
On more thoughts a [scheduled executor service](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ScheduledExecutorService.html) is better

Reply:
Currently, I followed the way this [TablePollerThread](https://github.com/confluentinc/kafka-connect-jdbc/blob/master/src/main/java/io/confluent/connect/jdbc/source/TableMonitorThread.java) is implemented for the `kafka-connect-jdbc`.

I can still implement the `ScheduledExecutorService` changes if required.